### ISSUE TITLE
Conditions Function Error based on  Multiple Selection

### ISSUE THREAD

**Issue by Jerrygu113** [directus/directus#22054]
### Describe the Bug

When I set the conditions on my field. if my condition is based on another multiple-select drop down list, It always doesn't work.  my drop dwon list is csv type

### To Reproduce

1. Create any field and set the condition like hidden in details 
2. set the rules with multiple drop down list
<img width="572" alt="github error" src="https://github.com/directus/directus/assets/132370852/f5476c8c-9253-445c-bb4b-83fca571a34f">
3. when I select type, it is not hidden as setting.

### Directus Version

v10.6.4

### Hosting Strategy

Directus Cloud

**Comment by paescuj** [comment]
You'll need to use the "contains" condition for CSV fields:

<img width="300" src="https://github.com/directus/directus/assets/5363448/b3b6a533-8b28-4bc7-a4c0-ed5cd5a8fb47">

Support for this type of condition for CSV fields has been added in #22002 which will be available in the next release.

Leaving this issue open for now, since we might want to improve the UX of the conditions interface
 - to display a dropdown with available options for "contains" condition, as it's currently the case with "equals"
 - maybe not remove the dropdown selection for "equals"

**Comment by Jerrygu113** [comment]
yeah, I have tried to use "contain" but it still doesn't work, I will wait the update of new version.
